core/dbt/events/types.py

```diff
@@ -2384,6 +2384,30 @@ def message(self) -> str:
         return "Internal event buffer full. Earliest events will be dropped (FIFO)."
 
 
+@dataclass
+class Sha1ChecksumPasses(DebugLevel):
+    code: str = "Z049"
```

Comment:
Love that you added some log lines!  Let's label these as type 'M' instead of 'Z' since it's related to deps generation.

Reply:
thanks for the direction here, made both of these changes